pkg/controllers/crdcompatibility/objectpruning/handle_test.go

```
@@ -362,8 +359,14 @@ var _ = Describe("Object Pruning Integration", func() {
 
 			By("Verifying error response")
 
-			err := cl.Create(ctx, testObj)
-			Expect(err).To(MatchError(ContainSubstring("CompatibilityRequirement.apiextensions.openshift.io \"non-existent-compat-req\" not found")))
+			Eventually(func(g Gomega) {
+				g.Expect(client.IgnoreNotFound(cl.Delete(ctx, testObj))).To(Succeed())
+				err := cl.Create(ctx, testObj)
```

golangci-lint (2.11.4)

[error] missing whitespace above this line (too many statements above if)

(wsl_v5)